### DISCUSSION TITLE
Can't login or signup...

### DISCUSSION THREAD

**Discussion by Yos666666** [vercel/community#5224] [Account Issue]
### Question

Hello,
I can't login or signup with my account.
I have always the same message :
"We are unable to complete your signup."
Previously, I tried to create my account directly through my Github account, but it said it was blocked!
My case number when contacting Vercel support : 00181611 if it can be useful...

Thank you.

**Comment by Yos666666** [comment]
It seems I'm not the only one this kind of thing happens to and I have the impression that Vercel support is doing nothing.
How is something so simple so complicated?

**Reply by AngelSifontes** [reply]
ami tambien

**Comment by Yos666666** [comment]
It's been more than 2 weeks now and I still have no response from Vercel and I despair at not moving forward on my project... if other people have the same problem or have solution, don't hesitate to write it here!

**Comment by amyegan** [comment]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns, so you did the right thing by opening that support case.

Tickets are processed by the order in which they are received, so it can take some time for the team to respond to all of the earlier cases that were opened before they reach yours. For more information about response times, please visit https://vercel.com/guides/vercel-support-queue-time

For more info regarding login issues, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Reply by Yos666666** [reply]
Thank you for your response... I understand that there is a waiting time, however for such a simple problem, I don't understand why it would take so long? (almost 1 month!)
I'm just worried about the future, some people haven't had a response after 2 months :-/

**Comment by xsayedmahmud** [comment]
I faced a similar issue where I couldn't verify my mobile number due to not receiving any verification code, despite attempting with various numbers. I reached out to Vercel support via email on December 16th last year, but there has been **no follow-up**. It seems like their response time might be faster for users with pro plans. However, how will i subscribe to a pro plan when I can't even log in initially. I've heard many positive feedback about Vercel support on Twitter, so this experience is quite unexpected.

**Reply by Yos666666** [reply]
I'm in the same situation, as I can't register, I can't do anything, it's quite frustrating.

**Reply by wfrm** [reply]
me too

**Comment by ghost** [comment]
Vercel is SUCK!
